### PR TITLE
MSI-824: Search in Low Stock Report grid is broken

### PR DESCRIPTION
**Pull request by p-bystritsky**
### Fixed Issues (if relevant)

1. magento-engcom/msi#824: Search in Low Stock Report grid is broken

### CODE REVIEW

File: app/code/Magento/Backend/view/adminhtml/templates/widget/grid.phtml [hunk 1]
```diff
@@ -48,6 +48,7 @@ $numColumns = sizeof($block->getColumns());
                 <?php else: ?>
                     <?= $block->getMainButtonsHtml() ? '<div class="admin__filter-actions">' . $block->getMainButtonsHtml() . '</div>' : '' ?>
                 <?php endif; ?>
+                <?php $block->getCollection()->load() ?>
```
Comment (naydav) [inline app/code/Magento/Backend/view/adminhtml/templates/widget/grid.phtml]: We don't need to load collection forcible

Comment (p-bystritsky) [reply app/code/Magento/Backend/view/adminhtml/templates/widget/grid.phtml]: @naydav we need to execute code located in _beforeLoad() method.

Comment (p-bystritsky) [reply app/code/Magento/Backend/view/adminhtml/templates/widget/grid.phtml]: @naydav  fixed.

**Comment by naydav** [comment]
Closed due to this PR contains not needed changes 

Also, this PR leads to new bug related to choosing of Store View

**Comment by naydav** [comment]
https://github.com/magento-engcom/msi/pull/840 has been created

**Pull request closed without merging**